### ISSUE TITLE
Undefined symbols for architecture arm64:   "_OBJC_CLASS_$_TWTRTwitter"

### ISSUE THREAD

**Issue by rmisos** [stewwan/capacitor-twitter#2]
Hi
I have followed the steps for the iOS Setup, but have run into this error:
`Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_TWTRTwitter", referenced from:
      objc-class-ref in Plugin-725b4cf30429efca35ac0c4dbabbe49ff07bd250f1c06da0ffa457e8c2bf325d.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)`

Ionic versions:
$ ionic info
Ionic:

   Ionic CLI                     : 5.2.3 (/usr/local/lib/node_modules/ionic)
   Ionic Framework               : @ionic/angular 4.9.1
   @angular-devkit/build-angular : 0.801.3
   @angular-devkit/schematics    : 8.1.3
   @angular/cli                  : 8.1.3
   @ionic/angular-toolkit        : 2.0.0

Capacitor:

   Capacitor CLI   : 1.2.1
   @capacitor/core : 1.2.1

Utility:

   cordova-res : not installed
   native-run  : not installed

System:

   NodeJS : v11.15.0 (/usr/local/bin/node)
   npm    : 6.7.0
   OS     : macOS Mojave

Any idea what's going wrong here?

**Comment by rmisos** [comment]
Seems like this is an Xcode thing.  I found a solution to it from this post:
[https://github.com/invertase/react-native-firebase/issues/321#issuecomment-321232473](url)
Different class but same error.

So basically I did this:
`rm -rf ~/Library/Developer/Xcode/DerivedData`
`cd ios/App`
`pod deintegrate && pod cache clean --all`
`sudo gem install cocoapods`
`pod repo remove master --verbose && pod setup --verbose && pod install --verbose`
or just
`pod setup --verbose && pod install --verbose`
then
`npx cap open ios`
and now it is working!  I think upgrading Cocoapods probably helped...

Thanks for this great plugin :)
